src/main/java/com/spring_boot/controllers/MailController.java

```diff
@@ -0,0 +1,28 @@
+package com.spring_boot.controllers;
+
+import com.home_work.spring_boot.entity.Letter;
+import com.home_work.spring_boot.services.MailService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "mails")
+public class MailController {
+
+    private MailService mailService;
+
+    @Autowired
+    public MailController(MailService mailService) {
+        this.mailService = mailService;
+    }
+
+    @RequestMapping(value = "saving-mail", method = RequestMethod.POST)
```

Comment:
according to rest principles, URL should represent the name of the entity/resource you want to work with. saving-mail doesn't sound like a name. What you are trying to do here is inserting a new letter object into collection of letters. To address collection of letters you can use `/letters`(or `/mails') as the name of the resource. The operation is expressed by http method, in this case POST means create/add. Finally, a URL will be `POST /letters`. This allows you to define different routes/endpoints for the same resource: for example you can add an endpoint `DELETE /letters` which will remove all the letters. The name of the resource haven't changed, it's still `/letters`. But the http method is telling what you want to do with a resource